docs/reference/announcements.md

```diff
@@ -108,6 +108,10 @@ An example of how to use the new `CorrelationResult` can be found in the [Connec
 
 The separated Ingress Helm configuration for Camunda 8 Self-Managed has been deprecated in 8.6, and will be removed from the Helm chart in 8.7. Only the combined Ingress configuration is officially supported. See the [Ingress guide](/self-managed/setup/guides/ingress-setup.md) for more information on configuring a combined Ingress setup.
 
+#### Helm chart - MultiRegion Failover / Failback mode deprecation
+
+The MultiRegion configuration in the helm chart has been deprecated in 8.6, and will be removed from the Helm chart in 8.7. This will be replaced with a set of API actuator endpoints that you would call while following the ([Dual-region operational procdure](/self-managed/operational-guides/multi-region/dual-region-ops.md))
```

Comment:
I'd go with `Multi-region` as we seem to align on that wording in the docs.

I'd also consider limiting the phrasing to the `global.multiregion.installationType` option as `global.multiregion.regionId` remains, otherwise it may sound like we fully remove mutli-region support.

Reply:
`installationType` was solely a Helm chart option to do some broker id magic and overwrite broker behaviour in case of running the operational procedure to failover / failback. The applications themselves don't know this option.

Reply:
yeah the current wording does sound scary. I've updated this.